resources/public/dashboard/index.html

```diff
@@ -204,9 +204,9 @@
        snag: getValueSnag},
       {description: "Command Queue",
        addendum: "depth",
-       url: "/metrics/v1/mbeans/org.apache.activemq:type=Broker,brokerName=localhost,destinationType=Queue,destinationName=puppetlabs.puppetdb.commands",
+       url: "/metrics/v1/mbeans/puppetlabs.puppetdb.mq%3Aname%3Dglobal.depth",
        format: d3.format(",s"),
-       snag: function(res) { return res["QueueSize"]; }},
+       snag: function(res) { return res["Count"]; }},
```

Comment:
Talked with Wyatt about this, this is changing anyway since the prefix isn't including the acivemq broker information, I'm good to go with this.